app/models/food.rb

```diff
@@ -0,0 +1,8 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_food
```

Comment:
**You guys did astonishing job!** I really like it!

- [ ] However, there are mini places that need to be filled up to make this app even greater. Let's get started.
In the requirement we can see that you are able to remove food from the list. But, unfortunately I was not able to remove any food from the list. Kindly make it work. I think you forgot to put `, dependent: :destroy` after `:recipe_food`. Kindly add it! Also, make sure the food is only removable by owner not from other users. 

```suggestion
    has_many :recipe_food, dependent: :destroy
```

Reply:
resolved